lib/codeshare_web/channels/user_socket.ex

```diff
@@ -18,10 +18,12 @@ defmodule CodeshareWeb.UserSocket do
   # To deny connection, return `:error`.
   #
   # See `Phoenix.Token` documentation for examples in
-  # performing token verification on connect.
-  def connect(_params, socket, _connect_info) do
+  # performing token verification on connect.%{"username"=>username}assign(socket, :username, "HARSH")
+  def connect(%{"username"=>username}, socket, _connect_info) do
```

Comment:
I think it would be better to not match the parameters being passed here and handle it seperately inside (for right now while we are still in development phase)